protocol/daemons/server/types/constants.go

```diff
@@ -10,8 +10,9 @@ const (
 	DaemonStartupGracePeriod = 60 * time.Second
 
 	// MaximumLoopDelayMultiple defines the maximum acceptable update delay for a daemon as a multiple of the
-	// daemon's loop delay.
-	MaximumLoopDelayMultiple = 3
+	// daemon's loop delay. This is set to 8 to have generous headroom to ignore errors from the liquidations daemon,
+	// which we have sometimes seen to take up to ~10s to respond.
+	MaximumLoopDelayMultiple = 8
```

Comment:
Updated this so that we can accomodate liquidations daemon runtimes of 10+ seconds without spurious error logs.